src/streaming/protection/Protection.js

```diff
@@ -96,6 +96,23 @@ const APIS_ProtectionModel_3Feb2014 = [
         message: 'mskeymessage',
         ready: 'mskeyadded',
         close: 'mskeyclose'
+    },
+    // webkit-prefixed (Safari, OS X)
+    {
+        // Video Element
+        setMediaKeys: 'webkitSetMediaKeys',
+        // MediaKeys
+        MediaKeys: 'webkitKeys',
```

Comment:
Surely this should be WebKitMediaKeys?

Reply:
Nope
https://developer.apple.com/documentation/webkitjs/htmlmediaelement/1633588-webkitkeys

Reply:
The linked page states that webkitKeys is a property of HTMLMediaElement, of type WebKitMediaKeys, `MediaKeys` needs to be a `new`-able constructor.

It's interesting that you have this working since, when you call `new window['webkitKeys']('com.apple.fps.2_0')`, as the player would, at least in the latest Safari TP you'll get a ReferenceError because it's not available. Whereas WebKitMediaKeys can be constructed and passed to webkitSetMediaKeys as expected.

Still, there are many more experienced people than me in terms of Safari DRM here, hopefully one of them can weigh in.